.github/workflows/ci.yaml

```diff
@@ -75,3 +75,9 @@ jobs:
       - run: |
           pip install tox
           tox -e docs
+
+  docker:
+    runs-on: ubuntu-20.04
```

Comment:
not latest?

Reply:
latest is randomly 18.04 or 20.04 currently (https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on)